coursefinder/templates/utils.py

```diff
@@ -0,0 +1,149 @@
+from CMS.test.mocks import StatusMocks
```

Comment:
Not an issue, but looks a little odd pulling the status codes from tests. Could maybe use the Python http module for these status codes in both areas?

Reply:
I concur

Reply:
Agree it started out only being used for the tests, need to change this.